test/e2e/play_kube_test.go

```diff
@@ -2475,6 +2475,35 @@ var _ = Describe("Podman kube play", func() {
 		Expect(logs.OutputToString()).To(Equal("podman"))
 	})
 
+	// If you have an init container in the pod yaml, podman should only create but not run the init container with kube play
+	// With annotation set to idle
+	It("test with init containers and annotation set to idle", func() {
+		// With the init container type annotation set to idle
+		pod := getPod(withAnnotation("io.podman.annotations.init.container.type", "idle"), withPodInitCtr(getCtr(withImage(CITEST_IMAGE), withCmd([]string{"printenv", "container"}), withInitCtr(), withName("init-test"))), withCtr(getCtr(withImage(CITEST_IMAGE), withCmd([]string{"top"}))))
```

Comment:
Does this mean that the annotation is at the pod level? If yes, is there a way to set different values to different init-containers?

Reply:
I don’t think Podman supports this annotation per container for type once and always.